onnxruntime/python/session.py

```diff
@@ -79,3 +80,26 @@ def end_profiling(self):
         :meth:`onnxruntime.SessionOptions.enable_profiling`.
         """
         return self._sess.end_profiling()
+
+    def to_onnx(self):
```

Comment:
if it's returning bytes[], why not call it **to_bytes()**? **to_onnx** sounds too mysterious -- unless it's doing some special processing besides returning byte representation.